examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<p>I would make the analogy to the BetaBinomial. The same way that the BB can be used to model overdispersed Binomial data (by assuming each batch of data follows a slightly different p, from a common Beta), the DM can be used to model overdispersed Multinomial data (by assuming each batch of data follows a slightly different p, from a common Dirichlet).</p><p> </p><p>One thing that has confused me in the past is that this "different" p applies for a batch of size n, and not for every trial observed. I used to think it was the latter and this was confusing for me. Not sure if others feel the same way.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>